Azure/azure-cli
# [AKS] `az aks create`: Add optional parameter `--revision` to set revision for the Azure Service Mesh addon while creating AKS cluster

**Related command**
`az aks create --enable-azure-service-mesh`

**Description**
Add optional parameter `--revision` to set revision for the Azure Service Mesh addon while creating AKS cluster.
Implicitly enable istio when ingress gateway is enabled for Azure Service Mesh.

**Testing Guide**
`az aks create -g MyResourceGroup -n MyManagedCluster --enable-azure-service-mesh --revision asm-1-18`
`az aks mesh enable-ingress-gateway -g MyResourceGroup -n MyManagedCluster`

This checklist is used to make sure that common guidelines for a pull request are followed.
